### ISSUE TITLE
Stale Issue Automation

### ISSUE THREAD

**Issue by mkaz** [WordPress/gutenberg#28900]
Thanks to everyone who provided feedback and contributed to the discussion around [the proposal posted on Make here](https://make.wordpress.org/core/2021/01/14/stale-issues-in-gutenberg-repository/). We’ve listened to all the ideas there and are proposing the following based on what was discussed:

* The bot will not close any issues. 

* After 6 months of no activity, the bot will help automatically label issues with `[Status] Needs confirmation`. If no action is taken after 1 month with this new label, the bot will then add `[Status] Stale`. This should help those triaging issues. 
 
* If any comment is made after the actions taken by the bot the labels added will be removed. 

The goal of the bot is to re-engage users by prompting them to confirm the problem is still valid with the latest versions.

For context, this is the proposed working message that the bot will apply as a comment after adding the `[Status] Needs confirmation` label to the issue itself:

> Hi @username,
> This ticket has gone 180 days without any activity and therefore meets the project’s requirement for a check-in to ensure it is not obsolete. If you are still experiencing this issue, then please reply with confirmation and any updated reproduction steps.
> Thanks,
> GutenBot

Related, in a parallel issue, we implemented a bot to enforce the Needs Info policy. The policy has been in place for a while, [see triage document](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/triage.md), but the bot to enforce was new ([see PR](https://github.com/WordPress/gutenberg/pull/28179)). I wanted to call it out after all the feedback on the post, it seems like we might want to adjust that policy and not have a bot enforce it.

**Comment by annezazu** [comment]
Thanks for doing such a great job of summarizing the discussion into actionable next steps. Excited to see this move forward! I want to offer up some feedback on the messaging in the form of two alternatives to consider:

Here's an option with simpler language/sentences:

> Hi @username,
This issue has gone 180 days without any activity. This meets the project’s requirement for a check-in to make sure it's still relevant. If you are still experiencing this issue, please reply to confirm and to include any updated reproduction steps.
Thanks,
GutenBot

Here's an option with more "personality" to make it less robotic (get it):

>Hi @username,
Oh no! This issue has gone 180 days without any activity. This means it's time for a check-in to make sure that what you found is relevant today. If you are still experiencing this issue, you can help the project by responding to confirm the problem and by providing any updated reproduction steps.
Thanks for helping out,
GutenBot

Perhaps instead of "Oh no!", we could go with something less intense like "Whoops!" or "Uh oh!". Could also be removed entirely but I kind of think it adds a nice touch of acknowledging that it's been a while since the issue was interacted with.

**Comment by DaisyOlsen** [comment]
Just a note to say that I like this approach as it avoids removing the human element and, instead, acts as a tool for those doing triage work to be more efficient.

**Comment by brentswisher** [comment]
I think this is a great start!  One thought on wording, having something like "Oh no!" or "Oops" seems to imply that it is the issue author's fault nothing is happening, which in many cases probably isn't accurate? Thinking of someone without the technical knowledge that submits an issue - after that, it really isn't up to them if something happens or not. Just a thought.

**Comment by annezazu** [comment]
Totally, I'm wondering what the best exclamation might be if we even want to go that route and agree it's something to get right. In my mind, it can help acknowledge how long it's been since the issue was addressed but definitely don't want it to seem as though it's the author's fault. Ideas welcome for other exclamations to use!

**Comment by skorasaurus** [comment]
Thanks @mkaz ; the proposed changes sounds great. As someone who triages, I appreciate that the issues aren't automatically closed as in my experience, most of the open issues that do not have any activity are still applicable. 

At the risk of the following being too wordy, perhaps offer a line that states: 

Are you able to still reproduce this issue using the [Gutenberg plugin](https://wordpress.org/plugins/gutenberg) that when enabled, offers the latest updates and features to Gutenberg/the block-editor ?
